ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/SignatureIntegrationForm.tsx

```diff
@@ -63,6 +92,31 @@ const VerificationExpandableSection = ({ toggleText, children }): ReactElement =
     );
 };
 
+function regularExpressionIcon(): ReactElement {
+    return (
+        <Popover
+            bodyContent={
+                <div>
+                    <a href="https://golang.org/s/re2syntax" target="_blank" rel="noreferrer">
+                        See RE2 syntax reference
```

Comment:
Absence of link style in popover is missed fix from PatternFly 5 upgrade. I will open separate PR.